### ISSUE TITLE
How to train?

### ISSUE THREAD

**Issue by niluanwudidadi** [novioleo/crnn.mxnet#7]
Hello, @novioleo ! Your work is very nice, I just use the demo，the result is good.But when I use my dataset(a picture contains two lines of characters), the result  is not satisfactory. So, can you give me some help for training(include preparing dataset, and parameter);or can you give me some example, I can't start this project!Thank you very much!

**Comment by novioleo** [comment]
This project not proposed for multiline text recognition,If you'd like to try,the simplest way is localize the text and extract it from image,then predict every line.if you want to a single model,i suggest you can try STN.
https://github.com/Bartzi/stn-ocr

this is so difficult to train and you need more priori knowledge in deep network.

**Comment by niluanwudidadi** [comment]
Again me, @novioleo. I have split my image into one line image, when I run train.py, occur this error:
/home/yao/mxnet-0.11.0/dmlc-core/include/dmlc/./logging.h:308: [14:48:57] src/operator/./slice_channel-inl.h:208: Check failed: dshape[real_axis] % param_.num_outputs == 0U (8 vs. 0) You are trying to split the 3-th axis of input tensor with shape (64,72,4,25) into num_outputs=17 evenly sized chunks, but this is not possible because 17 does not evenly divide 25

So I  set the seq_len=25, then it trained, and very faster, Time cost=0.550(400 images, is this too faster?);
But when I run predictor.py, error occur:
/home/yao/mxnet-0.11.0/dmlc-core/include/dmlc/./logging.h:308: [14:42:40] src/operator/./slice_channel-inl.h:208: Check failed: dshape[real_axis] % param_.num_outputs == 0U (7 vs. 0) You are trying to split the 3-th axis of input tensor with shape (1,72,4,32) into num_outputs=25 evenly sized chunks, but this is not possible because 25 does not evenly divide 32
How to solve this problem?
Thank you!

**Comment by novioleo** [comment]
@niluanwudidadi 你训练参数错了。你的图片宽度应该是字符数的4倍。你把的训练的脚本发出来我看下是不是错了。

**Comment by niluanwudidadi** [comment]
@novioleo 你好，很感谢你的帮助。我的训练脚本是python train.py --name wg --charset digit_letter.txt --train_lst data.lst --val_lst val.lst --seq_len 25 --num_label 62 --learning_rate 0.01 --gpu。我的图片大小是（h:265，w;58）,我的字符类别是：0123456789QWERTYUIOPASDFGHJKLZXCVBNMqwertyuiopasdfghjklzxcvbnm
可否提供一个联系方式，方便后续联系，谢谢。

**Comment by niluanwudidadi** [comment]
@novioleo 你好，很感谢你的帮助。我的训练脚本是python train.py --name digit_letter --charset digit_letter.txt --train_lst data.lst --val_lst val.lst --seq_len 25 --num_label 62 --learning_rate 0.01 --gpu。我的图片大小是（h:265，w;58）,我的字符类别是：0123456789QWERTYUIOPASDFGHJKLZXCVBNMqwertyuiopasdfghjklzxcvbnm
可否提供一个联系方式，方便后续联系，谢谢。
